test/ts-scenario/features/base_api_v2.feature

```diff
@@ -0,0 +1,66 @@
+	Then the request named myDiscoveryRequest for client fred has a general result matching {"result":"SUCCESS"}
+	And the request named myDiscoveryRequest for client fred has a event result matching {"result":"Commit success"}
+	And the request named myDiscoveryRequest for client fred has a commit result matching {"status":"SUCCESS"}
+
+Scenario: Using only fabric-common I can replay events on v2 channel
+	When I create an event service myEventService2 as client fred on channel eventschannel
+	And I regisister a block listener named myBlockListener with myEventService2 for startBlock 1 and endBlock 2 as client fred
+	And I regisister a chaincode listener named myChaincodeListener with myEventService2 for createCar event on contract fabcar as client fred
+	And I regisister a transaction listener named myTransactionListener with myEventService2 for all transactions as client fred
+	And I start the event service myEventService2 as filtered blocks to start at block 0 and end at block 6 as client fred
+	And I build a new endorsement request named replay3 for smart contract named fabcar with arguments [createCar,2006,Ford,Focus,blue,Henry] as client fred on channel eventschannel
+	And I commit the endorsement request named replay3 as client fred on channel eventschannel
+	Then the event listener myBlockListener of myEventService2 has results matching {"block":"2"} as client fred
+	Then the event listener myChaincodeListener of myEventService2 has results matching {"createCar":""} as client fred
+	Then the event listener myTransactionListener of myEventService2 has results matching {"transaction":"5"} as client fred
+
+Scenario: Using only fabric-common I can replay events on v2 channel
+	When I create an event service myEventService2 as client fred on channel eventschannel
+	And I regisister a block listener named myBlockListener with myEventService2 for startBlock 1 and endBlock 2 as client fred
+	And I regisister a chaincode listener named myChaincodeListener with myEventService2 for createCar event on contract fabcar as client fred
+	And I regisister a transaction listener named myTransactionListener with myEventService2 for all transactions as client fred
+	And I start the event service myEventService2 as full blocks to start at block 0 and end at block 7 as client fred
+	And I build a new endorsement request named replay4 for smart contract named fabcar with arguments [createCar,2005,Ford,Taurus,white,Henry] as client fred on channel eventschannel
+	And I commit the endorsement request named replay4 as client fred on channel eventschannel
+	Then the event listener myBlockListener of myEventService2 has results matching {"block":"2"} as client fred
+	Then the event listener myChaincodeListener of myEventService2 has results matching {"createCar":"PTCurser"} as client fred
+	Then the event listener myTransactionListener of myEventService2 has results matching {"transaction":"6"} as client fred
```

Comment:
Is this relying on other scenarios having run (and the order in which they run), including ones in feature files outside of this one? Expecting blocks 0 to 7 to exist and contain 6 transactions. If so, we might hit problems down the line if either tests change, just a subset of tests are run, or tests run in an unexpected order. Is there any practical alternative?

Reply:
these should be in the same scenario